_episodes/01-welcome.md

```diff
@@ -35,7 +35,7 @@ to conform to our [Code of Conduct]({{ site.coc }}). This Code of Conduct applie
 
 > Introductions set the stage for learning.
 > — Tracy Teal, Executive Director, The Carpentries
-{: .quotation}
+{: .testimonial}
```

Comment:
Does quotation not exist as a style for these blocks?

Reply:
Check the formatting of the page on GitHub - it's rendered as just a normal quotation, rather than a special box.
Available styles of special blockquotes are given here: http://carpentries.github.io/lesson-example/04-formatting/index.html#special-blockquotes